bin/middlewares/og_meta.js

```diff
@@ -80,8 +77,11 @@ module.exports = function(config) {
   return function(req, res, next) {
     const placeUrlMatch = req.originalUrl.match(/place\/(.*)/);
     const locale = res.locals.language;
+    let poiId;
     if (placeUrlMatch && placeUrlMatch.length > 0) {
-      const poiId = placeUrlMatch[1];
+      poiId = placeUrlMatch[1];
+    }
+    if (poiId && !poiId.startsWith('latlon:')) {
```

Comment:
this change is no longer necessary, now that Idunn API can handle `latlon:` places

Reply:
Fixed https://github.com/QwantResearch/erdapfel/pull/289/commits/6d7bb0eb8694653143ca059f006ec7d8aa5c62dd